requirements.in

```diff
@@ -18,7 +18,7 @@ django-user-agents==0.4.0
 djangorestframework==3.12.4
 djoser==2.1.0
 edx-api-client==0.10.0
-django-storages==1.7.1
+django-storages==1.11.1
```

Comment:
Is there any specif reason to not take this to the latest version of the storage available i.e. [1.12.3](https://pypi.org/project/django-storages/1.12.3/#history)?

Reply:
following the same version as we in  `mitxonline` , If you have recommendation to be on latest then let me know.

Reply:
thinking to move gradually on later stage.